pom.xml

```diff
@@ -22,7 +22,7 @@
 
   <groupId>io.yupiik.metrics</groupId>
   <artifactId>metrics-scraper</artifactId>
-  <version>1.0.4</version>
+  <version>1.0.5</version>
```

Comment:
Avoid updating pom version into issues commit please. It should be done during the release process.

Reply:
Noted for next time. I made the change